partner_financial_risk/models/res_partner.py

```diff
@@ -249,12 +250,24 @@ def process_unpaid_invoices(self):
         ConfigParameter = self.env['ir.config_parameter']
         last_check = ConfigParameter.get_param(
             'partner_financial_risk.last_check', default='2016-01-01')
-        move_lines = self.env['account.move.line'].search([
-            ('reconciled', '=', False),
-            ('account_id.internal_type', '=', 'receivable'),
-            ('date_maturity', '>=', last_check),
-            ('date_maturity', '<', max_date)])
-        move_lines.mapped('partner_id')._compute_risk_account_amount()
+        groups = self.env['account.move.line'].read_group(
```

Comment:
Shouldn't this be process with `sudo()`?

Reply:
Cron execs this process with admin user, but I don't have any problem to add sudo

Reply:
Being a public method that can be call from any place, I think it's convenient to be executed with `sudo`